src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/db/repositories/StaffRepository.kt

```diff
@@ -1,6 +1,9 @@
 package uk.gov.justice.digital.hmpps.hmppsstafflookupservice.db.repositories
 
+import kotlinx.coroutines.flow.Flow
 import org.springframework.data.repository.kotlin.CoroutineCrudRepository
 import uk.gov.justice.digital.hmpps.hmppsstafflookupservice.db.entities.Staff
 
-interface StaffRepository : CoroutineCrudRepository<Staff, Long>
+interface StaffRepository : CoroutineCrudRepository<Staff, Long> {
+  suspend fun findFirst20ByEmailLikeOrderByEmailAsc(email: String): Flow<Staff>
```

Comment:
What does this mean? `LikeOrderByEmailAsc`?! ..I expect it means `LikeEmail` but maybe spring data can't interpret that?